### ISSUE TITLE
Thunderbird Supernova 115.1.1 64-bit : Send-later column not available

### ISSUE THREAD

**Issue by Axolotl-13** [Extended-Thunder/send-later#602]
Like said in the title, with Thunderbird Supernova 115.1.1 64-bit, the Send-later column is not available.

With Thunderbird 102, the column exists, but is systematically unchecked: it can't be displayed permanently, so you have to check it again (and it disappears as soon as you change folder).

**Comment by jikamens** [comment]
> Like said in the title, with Thunderbird Supernova 115.1.1 64-bit, the Send-later column is not available.

Yes, this is documented all over the place, including in the [FAQ](https://github.com/Extended-Thunder/send-later/discussions/278) pinned at the top of the [discussions page](https://github.com/Extended-Thunder/send-later/discussions) and [in the user guide](https://extended-thunder.github.io/send-later/#the-send-later-column-in-the-drafts-folder-message-list-is-missing).

> With Thunderbird 102, the column exists, but is systematically unchecked: it can't be displayed permanently, so you have to check it again (and it disappears as soon as you change folder).

I am unable to reproduce this with Thunderbird 102.14.0 and Send Later 10.0.4. What versions do you have?
